images/manageiq-base/Dockerfile

```diff
@@ -45,18 +47,32 @@ RUN yum -y install https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-re
                    lvm2             \
                    net-tools        \
                    nmap-ncat        \
-                   nodejs           \
                    openldap-clients \
                    openscap-scanner \
                    patch            \
                    psmisc           \
-                   postgresql-devel \
                    sqlite-devel     \
+                   postgresql-devel \
                    sysvinit-tools   \
                    which            \
                    &&               \
     yum clean all
 
+# nvm environment variables
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 10.17.0
+# install nvm
+# https://github.com/creationix/nvm#install-script
+RUN curl --silent -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash
```

Comment:
If we do have to go this direction, is there a better way to install nvm? I'm not really a fan of `curl ... | bash`

Reply:
Yep, will replace this by installing node from here https://nodejs.org/dist/v10.18.1/